src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,13 +1,37 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
- */
 public class Bulldozer extends Machine {
+    private String bulldozerChassisType;
+    private String bladeType;
+    private String bulldozerType;
```

Comment:
Access modifiers are missing for the fields. Fields should be private to follow the principle of encapsulation.